fossunited/api/schedule.py

```
@@ -97,7 +121,411 @@ def get_event_schedule(event_id: str) -> dict:
+        tz = ZoneInfo(frappe.db.get_single_value("System Settings", "time_zone") or "Asia/Kolkata")
+        for s in sessions:
+            e = Event()
+            e.name = f"{s.title} - {doc.event_name}"
+            start = datetime.combine(s.scheduled_date, to_time(s.start_time), tzinfo=tz)
+            end = datetime.combine(s.scheduled_date, to_time(s.end_time), tzinfo=tz)
+            if end <= start:
+                continue
+            e.begin = start
+            e.end = end
+            e.location = f"{s.hall}, {metadata['Location']}"
+            e.description = f"Category: {s.category or 'N/A'}\nSpeakers: {', '.join([sp.get('name') for sp in s.speakers_list])}"  # noqa: E501
```

Ruff (0.12.2)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)